cockatrice/src/tappedout_interface.h

```diff
@@ -0,0 +1,28 @@
+#ifndef TAPPEDOUT_INTERFACE_H
+#define TAPPEDOUT_INTERFACE_H
+
+#include "carddatabase.h"
+#include "decklist.h"
+#include <QObject>
+
+class QByteArray;
+class QNetworkAccessManager;
+class QNetworkReply;
+class DeckList;
+
+class TappedOutInterface : public QObject {
```

Comment:
Class doc comments here would be good for other readers to know that this exists in order to support the "export to tappedout" feature.